### ISSUE TITLE
Panel title node should be configurable

### ISSUE THREAD

**Issue by muntact** [otissv/react-uikit-panel#1]
Kind of a feature request / issue. 

By default the panel title is put into an `</h3>` https://github.com/otissv/react-uikit-panel/blob/master/src/panel.jsx#L76

For proper semantics / accessibility the markup used for the title should be configurable, 
the uikit styles all `<hx>` markup the same for a panel header however they have different semantic meaning. 

However in my use case i am also putting the panel title inside a `<header>` so it might be nice to allow a react element to be set as the panel title. 

```
<header className="uk-panel-title uk-margin-remove uk-padding-remove">
  <h1 className="uk-margin-top uk-margin-right uk-margin-left">{title}</h1>
</header>
```

I can contribute a PR for the actual work at some point, was thinking that an issue would make it clear though :)
